app/src/main/java/com/example/wallet/MainActivity.kt

```diff
@@ -26,6 +29,33 @@ class MainActivity : AppCompatActivity() {
         setCurrentFragment(firstFragment)
 
         val bottomNavigationView: BottomNavigationView = findViewById(R.id.bottomNavigationView)
+        bottomNavigationView.setOnItemReselectedListener { item ->
+            when (item.itemId) {
+//                R.id.homeFragment -> firstFragment.view?.findViewById<RecyclerView>(R.id.upcoming_payment_View)?.scrollToPosition(0)
+//                R.id.chartFragment -> thirdFragment.view?.findViewById<RecyclerView>(R.id.upcoming_payment_chart_View)?.scrollToPosition(0)
```

Comment:
Does this comment code still use? If you don't use it anymore, please remove it.

Reply:
Done